joomla-projects/gsoc21_media-manager
# Week 5: Implement responsive images with user-specified sizes

Pull Request for issue #15.

### Summary of Changes
- Added a new option to the "Content - Responsive Images" plugin to override default image sizes.
- Implemented new form controls for form images to set custom sizes for each image separately.


### Testing Instructions
Go to System -> Plugins -> "Content - Responsive Images", set new responsive image sizes and save. Then insert an image to the web content and check it out on the front-end side - **srcset and sizes** attributes with the specified sizes will be added into your image.

![image](https://user-images.githubusercontent.com/62054743/125439799-cfbbe8ec-eb6b-4476-8eeb-f1dd8c0e6cc8.png)

What if you want to override the **new default** that you've specified for some images? This can be easily done using the "Custom responsive sizes" field in any form image options:

![image](https://user-images.githubusercontent.com/62054743/125440415-a23aee04-c334-42bb-8174-5006c36630b7.png)
